injection/sharedmain/main.go

```diff
@@ -255,6 +256,8 @@ func MainWithConfig(ctx context.Context, component string, cfg *rest.Config, cto
 	defer logger.Sync()
 	ctx = logging.WithLogger(ctx, logger)
 
+	klog.SetLogger(zapr.NewLogger(logger.Desugar()))
```

Comment:
I think this is a typo:
```suggestion
	klog.SetLogger(zap.NewLogger(logger.Desugar()))
```

Reply:
weird my import wasn't part of the commit - it should be zapr - let me push